src/service.ts

```
@@ -0,0 +1,77 @@
+    // Use provided launchId without starting a new one
+    if (this.options.launchId) {
+      process.env.RP_LAUNCH_ID = this.options.launchId;
+      this.isExternalLaunch = true;
+      return;
+    }
+
+    this.client = new RPClient(getClientConfig(this.options), getAgentInfo());
+
+    const launchObj = getStartLaunchObj(this.options, { name: this.options.launch });
+    const { tempId, promise } = this.client.startLaunch(launchObj);
+    
```

ESLint

[error] Delete `····`

(prettier/prettier)